internal/pkg/term/progress/progress.go

```diff
@@ -0,0 +1,19 @@
+// Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+// Package progress provides data and functionality to display updates to the terminal.
+package progress
+
+// Text is a description of the progress update.
+type Text string
+
+// Status is the condition of the progress update.
```

Comment:
Is it redundant to specify that these are defined by CFN?

Reply:
I was trying to go for Statuses that are agnostic of the type of deployment (cfn vs manual for example). Tbh though, right now they only work for CFN statuses :P

I think `HumanizeResourceEvents` needs a nicer interface that can take any piece of string (like a cfn status) and transform it to a type `progress.Status`.

Soon™️